src/client/components/HyfBigLogo/HyfBigLogo.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import HyfBigLogo from './HyfBigLogo';
+import { withKnobs } from '@storybook/addon-knobs';
+
+export default { title: 'HyfBigLogo Component', decorators: [withKnobs] };
+
+export const HyfLogo = () => {
+  return <HyfBigLogo title="WEB DEV ROOMS" />;
+};
```

Comment:
if you are not using knobs,, you can remove them

Reply:
Unused knobs removed